Exceptions/src/com/company/Divider.java

```diff
@@ -0,0 +1,45 @@
+package com.company;
+
+import com.company.Exceptions.InfiniteException;
+
+import java.util.Scanner;
+
+public class Divider {
+
+    public static void divDouble() throws InfiniteException {
+        Double val1 = 0.0;
+        Double val2 = 0.0;
+        Double result;
+        Scanner scanner = new Scanner(System.in);
+        boolean cycle = true;
+
+        do {
+            System.out.println("Please enter the value of the first variable: ");
+            try {
+                val1 = scanner.nextDouble();
+                cycle = false;
+            } catch (Exception exception) {
```

Comment:
catch particular exceptions, not general one

Reply:
@ltana, fixed